src/vector/gdal_to_geo.rs

```diff
@@ -110,3 +110,10 @@ impl TryFrom<Geometry> for geo_types::Geometry<f64> {
         }
     }
 }
+
+impl TryFrom<Geometry> for geo_types::Geometry<f64> {
+    type Error = GdalError;
+    fn try_from(value: Geometry) -> Result<Self, Self::Error> {
+        TryFrom::<&Geometry>::try_from(&value)
```

Comment:
```suggestion
        Self::try_from(&value)
```

Reply:
Is my suggestion somehow wrong?

I'm a bit confused, you :+1:ed it, marked it as resolved, but didn't change the code.

Reply:
@lnicola I forgot to `push` 🤦 . I'm recovering from _The Covid_, and not only forgot, but did them in the wrong order.  My apologies.

Reply:
No worries, and good luck with your recovery!